argus/client/sct/client.py

```diff
@@ -152,7 +152,7 @@ def submit_performance_results(self, performance_results: PerformanceResultsRequ
         )
         self.check_response(response)
 
-    def create_resource(self, name: str, resource_type: str, public_ip: str, private_ip: str,
+    def create_resource(self, name: str, resource_type: str, public_ip: str, private_ip: str, instance_type: str,
```

Comment:
SCT side should also be updated to pass this parameter as it is mandatory

Reply:
true, fixing that